### PR TITLE
Wording Changes for Calendar Baseline per Issue 135

### LINKED ISSUE #135: Changes to Calendar Baseline Wording
The following changes are needed to the calendar baseline:
- In CALENDAR.4.1v0.1, the following wording needs to be added as a note: "unless agency mission fulfillment requires collaboration between users internal and external to an organization who use both Microsoft Exchange and Google Calendar."
- Remove the following wording from the CALENDAR.3.1v0.1: "to restrict information sharing and prevent data leakage" from the policy statement, this belongs in the rationale."
- Remove the following wording from the CALENDAR.1.1v0.1: "to restrict information sharing and prevent data leakage" from the policy statement, this belongs in the rationale."

### PR DESCRIPTION
**Pull request by jkaufman-mitre**
Changed wording to the Calendar baseline in 4.1, 3.1, and 1.1. This PR addresses the comments in issue 135.

Fixes #135

### CODE REVIEW

File: baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ This section determines what information is shared from primary calendars with e
 ### Policies
 
 #### GWS.CALENDAR.1.1v0.1
-External Sharing Options for Primary Calendars SHALL be configured to "Only free/busy information (hide event details)" to restrict information sharing and prevent data leakage.
+External Sharing Options for Primary Calendars SHALL be configured to "Only free/busy information (hide event details)".
```
Comment (tmcomeau) [inline baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md:40]:
```suggestion
External Sharing Options for Primary Calendars SHALL be configured to "Only free/busy information (hide event details)."
```

File: baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ This section determines what information is shared from secondary calendars with
 ### Policies
 
 #### GWS.CALENDAR.3.1v0.1
-External sharing options for secondary calendars SHALL be configured to "Only free/busy information (hide event details)" to restrict information sharing and prevent data leakage.
+External sharing options for secondary calendars SHALL be configured to "Only free/busy information (hide event details)".
```
Comment (tmcomeau) [inline baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md:130]:
```suggestion
External sharing options for secondary calendars SHALL be configured to "Only free/busy information (hide event details)."
```

File: baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md [hunk 3]
```diff
@@ -181,11 +181,13 @@ Due to the added complexity and attack surface associated with configuring Calen
 ### Policies
 
 #### GWS.CALENDAR.4.1v0.1
-Calendar Interop SHOULD be disabled unless agency mission fulfillment requires collaboration between users internal and external to an organization who use both Microsoft Exchange and Google Calendar.
+Calendar Interop SHOULD be disabled.
 
 - Rationale
   - Prevent information exchange between Microsoft and Google calendars for users using both services, unless it is required by the organization.
```
Comment (tmcomeau) [inline baselines/Google Calendar Minimum Viable Secure Configuration Baseline v0.1.md:187]:
```suggestion
  - Minimize attack surface by not enabling this feature which relies on Exchange Web Services for information exchange between Microsoft and Google calendars, unless required by the organization.
```

**Review by tmcomeau — COMMENTED** [review]
Minor suggested edits. Please resend when ready and will review and then tag in @mitchelbaker-cisa

**Review by tmcomeau — APPROVED** [review]

**Review by adhilto — APPROVED** [review]

**Pull request merged**